examples/fillstates.py

```diff
@@ -80,18 +89,59 @@
         # calling colormap with value between 0 and 1 returns
         # rgba value.  Invert color range (hot colors are high
         # population), take sqrt root to spread out colors more.
-        colors[statename] = cmap(1.-np.sqrt((pop-vmin)/(vmax-vmin)))[:3]
+        colors[statename] = cmap(np.sqrt((pop-vmin)/(vmax-vmin)))[:3]
     statenames.append(statename)
-# cycle through state names, color each one.
+
+#%% ---------  cycle through state names, color each one.  --------------------
+fig = plt.gcf()  # get current figure instance
```

Comment:
The original example script has `ax = plt.gca()` on that location, and I just added `fig = plt.gcf()` on top of it because I wanted the changed to the original script to be minimal.

I can change it to what you suggested. It is indeed more elegant.